src/components/Modal/Modal.tsx

```diff
@@ -0,0 +1,97 @@
+"use client";
+
+import React, { ReactNode } from "react";
+import styled from "styled-components";
+import { Dialog } from "@mui/material";
+import CloseIcon from "@mui/icons-material/Close";
+
+const StyledDialog = styled(Dialog)`
+    & .MuiPaper-root {
+        border: 0;
+        border-radius: 2em;
+        padding: 0;
+        margin: 5%;
+        overflow: hidden;
+        max-width: 90%;
+    }
+`;
+
+const Header = styled.div`
+    display: flex;
+    position: sticky;
+    justify-content: space-between;
+    align-items: center;
+    color: ${(props) => props.theme.secondaryForegroundColor};
+
+    // TODO VFB-16 Add the equivalent of this colour to a palette without the transparency
+    background-color: ${(props) => props.theme.secondaryBackgroundColor}A0;
+
+    font-size: 1.5em;
+    font-weight: bolder;
+    padding: 0.75em 1em;
+    width: 100%;
+`;
+
+const Content = styled.div`
+    overflow: auto;
+    padding: 1em 1.5em 1.5em;
+    width: 100%;
+`;
+
+const CloseButton = styled.button.attrs({
```

Comment:
is a closed button common enough to be it's own file? potentially under Button?

Reply:
At this time, I don't think it's worth it at this point for the close button to be its own file just yet. Do you still want me to refactor it out into its own component?